gateway/api/rest-impl/src/main/java/io/apiman/gateway/api/rest/impl/IPlatformAccessor.java

```diff
@@ -31,5 +31,5 @@
      * Gets the platform.
      * @return the engine
      */
-    public IPlatform getPlatform();
+    IPlatform getPlatform();
```

Comment:
I know public is redundant, but I like it.  :)  Unless there's a compelling reason to remove it I'd prefer to leave it alone.